src/main/java/org/minions/devfund/yuri/movieDataBase/Movie.java

```diff
@@ -0,0 +1,75 @@
+package org.minions.devfund.yuri.movieDataBase;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a Movie with all attributes and behavior.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor method.
+     *
+     * @param name   movie name.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<>();
+    }
+
+    /**
+     * Constructor method by default.
+     */
+    public Movie() {
+        this("");
+    }
+    /**
+     * Gets actor list.
+     * @return actors list.
+     */
+    public ArrayList<Actor> getActors() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)